manifests/jetty.pp

```diff
@@ -50,26 +52,30 @@
   file { '/var/log/jetty':
     ensure => 'link',
     target => "${jetty_home}/jetty/logs",
-  } ->
-
-  file_line { 'jetty_startup_minutes':
-    ensure => present,
-    path   => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
-    line   => "    sleep ${jetty_start_interval}",
-    match  => '^    sleep \d+',
-  } ->
-
-  file { '/etc/init.d/jetty':
-    ensure => 'link',
-    target => "${jetty_home}/jetty-distribution-${jetty_version}/bin/jetty.sh",
   }
 
   if $::service_provider == 'systemd' {
     systemd::unit_file { 'jetty.service':
       content => template("${module_name}/jetty/jetty.service.erb"),
-      require => File['/etc/init.d/jetty'],
       before  => Service['jetty'],
```

Comment:
I was more concerned with changes to the unit file. For example, you update the contents of the unit file, then you want the service to be restarted, so your changes can take effect. A notify would do this for you.